third_party/blink/renderer/bindings/core/v8/local_window_proxy.cc

```diff
@@ -297,7 +297,9 @@ void LocalWindowProxy::Initialize() {
 
   // Add an event listener for the dispatched custom event the devtools uses to register
   // its listener.  Do this outside the recording.
-  SetupRecordReplayEventListener();
+  if (getenv("CHROMIUM_UI")) {
+    SetupRecordReplayEventListener();
```

Comment:
Can we rename this to `InitializeReplayChromiumUI` or somesuch?

Reply:
hm..  it's not really anything to do with UI itself, it's a communication channel.  How about `SetupRecordReplayDevtoolsComms`, or maybe `SetupRecordReplayWebChannel`?.  "WebChannel" shows up in the event names + the names the devtools code uses to send/receive messages, and might make it easier to recognize for folks who know both sides.

Reply:
#1032 actually pulls the event listener out into a separate file and fleshes it out - the resulting files could be renamed to `record_replay_web_channel.{cc,h}` from `record_replay_event_listener.{cc,h}` with all the communication code living there.